WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -60,93 +61,95 @@
-        <controls:ExtendedListBox Items="{Binding Coins}" VirtualizationMode="None" SelectedItem="{Binding Path=SelectedCoin, Mode=TwoWay}">
-          <controls:ExtendedListBox.ContextMenu>
-            <ContextMenu>
-              <!--Enqueuing needs password, TODO: jump to password box OR display pw box in context menu + send button-->
-              <MenuItem  Command="{Binding EnqueueCoin}" IsVisible="false">
-                <MenuItem.Header>
-                  <Grid>
-                    <Grid.ColumnDefinitions>
-                      <ColumnDefinition />
-                      <ColumnDefinition />
-                    </Grid.ColumnDefinitions>
-                    <TextBlock Text="Enqueue Coin, password:" VerticalAlignment="Center" />
-                    <TextBox Width="50" Grid.Column="1" />
-                  </Grid>
-                </MenuItem.Header>
-              </MenuItem>
-              <MenuItem Header="Dequeue from CoinJoin" Command="{Binding DequeueCoin}">
-                <MenuItem.Icon>
-                  <Path HorizontalAlignment="Left"
-                      Data="M3.24,7.51c-0.146,0.142-0.146,0.381,0,0.523l5.199,5.193c0.234,0.238,0.633,0.064,0.633-0.262v-2.634c0.105-0.007,0.212-0.011,0.321-0.011c2.373,0,4.302,1.91,4.302,4.258c0,0.957-0.33,1.809-1.008,2.602c-0.259,0.307,0.084,0.762,0.451,0.572c2.336-1.195,3.73-3.408,3.73-5.924c0-3.741-3.103-6.783-6.916-6.783c-0.307,0-0.615,0.028-0.881,0.063V2.575c0-0.327-0.398-0.5-0.633-0.261L3.24,7.51 M4.027,7.771l4.301-4.3v2.073c0,0.232,0.21,0.409,0.441,0.366c0.298-0.056,0.746-0.123,1.184-0.123c3.402,0,6.172,2.709,6.172,6.041c0,1.695-0.718,3.24-1.979,4.352c0.193-0.51,0.293-1.045,0.293-1.602c0-2.76-2.266-5-5.046-5c-0.256,0-0.528,0.018-0.747,0.05C8.465,9.653,8.328,9.81,8.328,9.995v2.074L4.027,7.771z" Fill="#22B14C"
-                      Height="16" Width="16"  Stretch="Fill" />
-                </MenuItem.Icon>
-              </MenuItem>
-            </ContextMenu>
-          </controls:ExtendedListBox.ContextMenu>
-          <controls:ExtendedListBox.ItemTemplate>
-            <DataTemplate>
-              <Grid>
-                <Expander Name="coinExpander" ExpandDirection="Down" Classes="coloredExpander"  Background="{Binding ElementName=coinExpander, Path=IsExpanded, Converter={StaticResource CoinItemExpanderColorConverter}}">
-                  <Grid HorizontalAlignment="Left" ColumnDefinitions="150, *" RowDefinitions="Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto,Auto" MaxWidth="800" Margin="35 10 0 25">
-                    <TextBlock Text="Transaction Id:" Grid.Row="0" />
-                    <controls:ExtendedTextBox Classes="selectableTextBlock Transaparent" Text="{Binding TransactionId}" Background="Transparent" Grid.Column="1" Grid.Row="0" />
+        <controls:BusyIndicator IsBusy ="{Binding FiltersLeft, Converter={StaticResource ShouldDisplayValueConverter}}" Text="Downloading filters...">
```

Comment:
Stray space in line 64: `IsBusy ="` -> ` IsBusy="`

Reply:
There are many inconsistent spacing on attributes on this file, which is not caused by this PR but perhaps we could fix those here?

Reply:
Yeah, I can go through and clean it up

Reply:
I think I caught all of them LMK if you find any that I missed